tests/Functional/Driver/OCI8/ConnectionTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional\Driver\OCI8;
+
+use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Exception\DriverException;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Tests\TestUtil;
+use Throwable;
+
+class ConnectionTest extends FunctionalTestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        if (TestUtil::isDriverOneOf('oci8', 'pdo_oci')) {
+            return;
+        }
+
+        self::markTestSkipped('This test requires the oci8 or pdo_oci driver.');
+    }
+
+    public function testPrepareThrowsErrorOnConnectionLost(): void
+    {
+        $this->markConnectionNotReusable();
+        $this->killCurrentSession();
+
+        $this->expectException(DriverException::class);
+
+        $this->connection->executeQuery('SELECT * FROM 1');
+    }
+
+    /**
+     * Kill the current session, by using another connection
+     * Oracle doesn't allow you to terminate the current session, so we use a second connection
+     */
+    private function killCurrentSession(): void
+    {
+        $row = $this->connection->fetchNumeric(<<<'SQL'
+        SELECT SID, SERIAL#
+        FROM V$SESSION
+        WHERE AUDSID = USERENV('SESSIONID')
+SQL);
```

Comment:
This should be indented as well. Otherwise, all lines above will have be indented, which is likely not what we want.

Reply:
Wasn't entirely sure what you meant here, so I've just changed it to match the style in getCreateTableSQL (src/Schema/SQLiteSchemaManager.php)